packages/ses/src/enable-property-overrides.js

```diff
@@ -18,6 +18,8 @@ function isObject(obj) {
   return obj !== null && typeof obj === 'object';
 }
 
+export const originalValueSymbol = Symbol.for('originalValue');
```

Comment:
Even exported, the symbol is not user-facing since we don’t reexport it from ses.js or lockdown.js (we don’t reexport anything any more on principle). Probably sufficient to test and instruct users to obtain the registered symbol only.

Reply:
I stopped exporting it and added a comment about how to obtain the registered symbol.